version: '3.8'

services:

  odoo:
    container_name: odoo
    image: odoo:18.0
    environment:
      HOST: spirally-masculine-sloth.data-1.use1.tembo.io
      USER: ${PG_USER}
      PASSWORD: superodoopassword
    ports:
      - 8069:8069
    volumes:
      - config:/etc/odoo
      - extra-addons:/mnt/extra-addons
      - data:/var/lib/odoo
    restart: always

  backup:
    image: offen/docker-volume-backup:latest
    restart: always
    environment:
      AWS_ENDPOINT: "s3.filebase.com"
      AWS_S3_BUCKET_NAME: "impressive-test-backup"
      AWS_ACCESS_KEY_ID: "cGy4W0jg3eDP3EVacZ7fbauULRF335IyhcC9Y4kr"
      AWS_SECRET_ACCESS_KEY: "F2AAC546FEB0F0C379D9"
      AWS_ENDPOINT_PROTO: "https"
    volumes:
      - data:/backup/my-app-backup:ro
      - extra-addons:/backup/extra-addons:ro
      - config:/backup/my-config-backup:ro

volumes:
  config:
  extra-addons:
  data:
